ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -226,66 +242,160 @@ ColumnLayout {
-                Layout.bottomMargin: Style.current.halfPadding
-                visible: d.hasCollectibles && d.hasCommunityCollectibles
+            Binding {
+                target: communityCollectiblesView
+                property: "header"
+                when: d.onlyOneType
+                value: null
+
+                restoreMode: Binding.RestoreBindingOrValue
             }
 
-            RowLayout {
-                Layout.fillWidth: true
-                Layout.leftMargin: Style.current.padding
-                Layout.rightMargin: Style.current.smallPadding
-                Layout.bottomMargin: 4
-                visible: d.hasCommunityCollectibles
-                StatusBaseText {
-                    text: qsTr("Community collectibles")
-                    color: Theme.palette.baseColor1
-                }
-                Item { Layout.fillWidth: true }
-                StatusFlatButton {
-                    Layout.preferredWidth: 32
-                    Layout.preferredHeight: 32
-                    icon.name: "info"
-                    textColor: Theme.palette.baseColor1
-                    horizontalPadding: 0
-                    verticalPadding: 0
-                    onClicked: Global.openPopup(communityInfoPopupCmp)
+            width: doubleFlickable.width
+            cellHeight: d.communityCellHeight
+
+            model: communityModel
+        }
+
+        flickable2: CustomGridView {
+            id: regularCollectiblesView
+
+            header: Item {
+                id: nonCommunityHeader
+
+                height: d.headerHeight
+                width: doubleFlickable.width
+
+                HeaderDelegate {
+                    width: nonCommunityHeader.width
+                    height: nonCommunityHeader.height
+
+                    parent: doubleFlickable.contentItem
+                    y: doubleFlickable.gridHeader2YInContentItem
+                    z: 1
+
+                    text: qsTr("Others")
+
+                    checked: doubleFlickable.flickable2Folded
+                    scrolled: (doubleFlickable.contentY >
+                               communityCollectiblesView.contentHeight
+                               - d.headerHeight)
+                    showInfoButton: false
+
+                    onToggleClicked: doubleFlickable.flip2Folding()
                 }
             }
 
-            CustomGridView {
-                id: communityCollectiblesView
-                cellHeight: d.communityCellHeight
-                model: CustomSFPM { isCommunity: true }
+            Binding {
+                target: regularCollectiblesView
+                property: "header"
+                when: d.onlyOneType
+                value: null
+
+                restoreMode: Binding.RestoreBindingOrValue
             }
+
+            width: doubleFlickable.width
+            cellHeight: d.cellHeight
+
+            model: nonCommunityModel
+        }
+    }
+
+    component HeaderDelegate: Rectangle {
```

Comment:
Would be nice to have this HeaderDelegate as a shared component (so that the ManageHiddenPanel.qml can reuse it too)

Reply:
Yeah, definitely it should be. But I would wait a bit with that. There are several variants in several places, everywhere a bit different. I believe that when having all of them in place it will be easier to exclude the common part nicely without overparametrizing the component.